docs/src/reference/python/operations/logic/allclose_example.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import equistore, numpy as np
+from equistore import TensorBlock, TensorMap, Labels, sum_over_samples
+from equistore import allclose, allclose_raise, allclose_block, allclose_block_raise
+
+# Create some TensorBlocks
+
+# Create simple block1
+block1 = TensorBlock(
+    values=np.array([
+         [1, 2, 4],
+         [3, 5, 6],
+     ]),
+    samples=Labels(
+         ["structure", "center"],
+         np.array([
+             [0, 0],
+             [0, 1],
+         ]),
+     ),
+    components=[],
+    properties=Labels(
+        ["properties"], np.array([[0], [1], [2]])
+     ),
+)
+keys = Labels(names=["key"], values=np.array([[0]]))
+
+# Recreate block1, but rename properties label 'properties' to 'PROPERTIES'
+block2 = TensorBlock(
+    values=np.array([
+         [1, 2, 4],
+         [3, 5, 6],
+     ]),
+    samples=Labels(
+         ["structure", "center"],
+         np.array([
+             [0, 0],
+             [0, 1],
+         ]),
+     ),
+    components=[],
+    properties=Labels(
+        ["PROPERTIES"], np.array([[0], [1], [2]])
+     ),
+)
+
+allclose_block(block1, block2)
+#Output:
+# False
+### This fails because allclose_block checks that the properties of two blocks are equal too
```

Comment:
here I dont have a strong opinion, but maybe calling them `property_1` and `property_2` is easier to understand rather than one is upper case of the other and that is why they are different.

Reply:
Bon idée